ansible/roles/configure_ib/tasks/main.yml

```diff
@@ -1,12 +1,42 @@
 ---
+- name: Make config drive mount directory
```

Comment:
In a world where config drive & IB works, is this required?

Reply:
No.